app/controllers/Application.scala

```diff
@@ -1,12 +1,194 @@
 package controllers
 
+import play.api.Play.current
 import play.api._
+import play.api.libs.json.Json.JsValueWrapper
 import play.api.mvc._
+import play.api.db._
+import play.twirl.api.Html
+import play.api.libs.json._
+
+import scala.collection.mutable.ArrayBuffer
 
 class Application extends Controller {
 
   def index = Action {
-    Ok  }
+    Ok
+  }
+
+
+  def listAgents = Action {
+    var jsonBuffer = ArrayBuffer.empty[JsValue]
+    val conn = DB.getConnection()
+    try {
+      val stmt = conn.createStatement
+      val rs = stmt.executeQuery("SELECT id, first_name, last_name, user_name from AGENT")
+      while (rs.next()) {
+        val json: JsValue = Json.obj(
+          "id" -> rs.getInt("ID"),
+          "firstName" -> rs.getString("first_name"),
+          "lastName" -> rs.getString("last_name"),
+          "userName" -> rs.getString("user_name")
+        )
+        jsonBuffer += json
+      }
+      rs.close()
+      stmt.close()
+    }
+    finally {
+      conn.close()
+    }
+    val j = JsArray(jsonBuffer)
+    Ok(j)
+  }
+
+
+  def validateAgent() = Action {
+
+    Ok
+  }
+
+
+  def addAgent() = Action {
+
+    Ok
+  }
+
+
+  //TODO we want to take some parameters for this call
+  // - Office location
+  // - Agent ID for tickets assigned to that agent
+  // - can these be
+  def listTickets = Action {
```

Comment:
I think at some point we need to factor out the redundancy between all these controller actions (and get rid of `var`s for bonus points), but I'm not sure if it should wait... (depends on the עולם's readiness...)